@xen-orchestra/xva-generator/toOvaXml.mjs

```diff
@@ -0,0 +1,38 @@
+// from package xml-escape
+function escape(string) {
+  if (string === null || string === undefined) return
+  if (typeof string === 'number') {
+    return string
+  }
+  if (string.startsWith('<dateTime.iso8601>')) {
```

Comment:
Don't do this, it opens door to injection.

If you want an escape hatch, do it higher when you no that the value is not user data.